frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -197,11 +259,32 @@ export class ImportWizard extends Wizard {
     });
   }
 
-  async import(config: VMImportConfig) {
+  async addVmNetworks(networkResources: NetworkResource[]) {
+    // Networking
+    // First update imported network interfaces to comply with k8s
+    await this.updateImportedNICs();
```

Comment:
I wouldn't call updateImportedNICs here.

Reply:
I didn't change anything there, calling updateImportedNICs was here already, the only thing is that I've moved all that part to a separate method addVmNetworks

Reply:
I agree with @rhrazdil, let's call it outside of this method to better split the logic. Plus I would not expect such side effects from a method called addVmNetworks

Reply:
Done, now calling of `updateImportedNICs` is done separately from import method